app/graphql/types/query_type.rb

```diff
@@ -17,15 +17,15 @@ class QueryType < Types::BaseObject
     field :users, [Types::UserType], null: false,
       description: "return all users"
 
-    field :user, Types::UserType, null: false do
-      description "return a user"
-      argument :id, ID, required: true
-    end
-
     def users
       User.all
     end
 
+    field :user, Types::UserType, null: false do
```

Comment:
what prompted moving this? would love to know more.

Reply:
IDK just wanted to make it match all the other code written in this file